bin/new_plugin_repo.rb

```diff
@@ -39,7 +37,5 @@
 puts
 puts "******* MANUAL THINGS *******"
 puts "- https://codeclimate.com/github/#{repo.github_repo} => Repo Settings => GitHub => Pull Request Status Updates => Install"
-puts "- https://hakiri.io and follow the new project."
 puts "- https://gitter.im/ManageIQ#createroom and create a new room linked to the repository"
-puts "- Add repo to https://github.com/ManageIQ/manageiq-release/blob/master/config/repos.yml"
```

Comment:
This is removed because we have a check for this at the beginning of this script and it won't continue unless that check passes.